R/system.R

```diff
@@ -2953,3 +2953,21 @@ filter_cascade <- function(.data, ...){
   }
   df
 }
+
+#
+#'@export
+load_fred <- function(series_id, date_start = "", date_end = NULL, password) {
```

Comment:
Why is date_start default is empty string while the date_end default is NULL?

Reply:
The default for date_end should be an empty string so I updated the default as "".